host/xtest/regression_6000.c

```diff
@@ -1477,46 +1478,50 @@ ADBG_CASE_DEFINE(regression, 6010, xtest_tee_test_6010, "Test Storage");
 #ifdef WITH_GP_TESTS
 static void xtest_tee_test_6011(ADBG_Case_t *c)
 {
-    TEEC_Session sess;
-    uint32_t orig;
-    /*
-     * Test data from
-     * Invoke_InitObjectAndAttributes_TEE_TYPE_AES_success_attribute_
-     * TEE_ATTR_SECRET_VALUE_correct_size (9d-9a-91)
-     */
-    static const uint8_t attr_meta[] = {
+	TEEC_Session sess = { };
+	uint32_t orig = 0;
+	/*
+	 * Test data from
+	 * Invoke_InitObjectAndAttributes_TEE_TYPE_AES_success_attribute_
+	 * TEE_ATTR_SECRET_VALUE_correct_size (9d-9a-91)
+	 */
+	static const uint8_t attr_meta[] = {
 0xc0,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x20,0x00,0x00,0x00,
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
-    };
-    static const uint8_t attr_data[] = {
+	};
+	static const uint8_t attr_data[] = {
 0x60,0x3d,0xeb,0x10,0x15,0xca,0x71,0xbe,0x2b,0x73,0xae,0xf0,0x85,0x7d,0x77,
 0x81,0x1f,0x35,0x2c,0x07,0x3b,0x61,0x08,0xd7,0x2d,0x98,0x10,0xa3,0x09,0x14,
 0xdf,0xf4,
-    };
+	};
 
-    if (!ADBG_EXPECT_TEEC_SUCCESS(
-            c, xtest_teec_open_session(
-                &sess, &gp_tta_ds_uuid, NULL, &orig)))
-        return;
+	if (!ADBG_EXPECT_TEEC_SUCCESS(
+			c, xtest_teec_open_session(&sess, &gp_tta_ds_uuid,
+						   NULL, &orig)))
+		return;
 
-    if (!ADBG_EXPECT_TEEC_SUCCESS(c, ds_init_object_and_attributes(&sess,
-            0xa0000010, 0x100, attr_meta, sizeof(attr_meta), attr_data,
-            sizeof(attr_data), 0)))
-        goto exit;
+	if (!ADBG_EXPECT_TEEC_SUCCESS(
```

Comment:
I'm not so sure this reformatting is an improvement.

Reply:
ok, will propose another.